examples/controls_example/public/edit_example.tsx

```diff
@@ -177,10 +177,12 @@ export const EditExample = () => {
             </EuiButton>
           </EuiFlexItem>
         </EuiFlexGroup>
+        {/* Can't wrap the ControlGroupRenderer in the `EuiSkeletonTitle` component like it's
+             upposed to be used since it's the one that sets loading to false. */}
```

Comment:
Just adding my 2c - this is a totally valid use case of the skeleton components just btw! While they can act as wrappers with conditional `isLoading` state as a nice syntactical sugar shortcut, you can also just use them as standalone `<EuiSkeleton />` components wrapped in conditional logic 🎊

Reply:
Ooh, good to know, thanks! I removed that comment in 6484c7266ebbb46bdbdec85d88adb12e5b2a8f46 💃